websockets/framing.py

```diff
@@ -182,9 +188,9 @@ def check_frame(frame):
     if frame.rsv1 or frame.rsv2 or frame.rsv3:
         raise WebSocketProtocolError("Reserved bits must be 0")
 
-    if frame.opcode in (OP_CONT, OP_TEXT, OP_BINARY):
+    if frame.opcode in [OP_CONT, OP_TEXT, OP_BINARY]:
```

Comment:
What about a `set` here (and below) instead? As far as perf goes, tuple creation is somewhat faster than that of lists so I would suggest keeping the original.